SDL/files/SDL_dcevents.c

```diff
@@ -101,7 +101,7 @@ static void mouse_update(void) {
 }
 
 static void keyboard_update(void) {
-    static kbd_state_t old_state;
+	static kbd_mods_t old_mods;
```

Comment:
```suggestion
    static kbd_mods_t old_mods;
```

It seems they're using spaces 😉